### DISCUSSION TITLE
shadow drone

### DISCUSSION THREAD

**Discussion by ferdiros** [bitcraze/discussions#88] [Q&A]
Hello I would like to check the accelerations of a drone in flight through the roll, pitch and Yaw coordinates of a drone on my wrist, I was able to log and write a program that translates the coordinates of drone 1 into accelerations. 
Now the problem is I don't know how to log from drone 1 and at the same time send the data to drone 2 which it has to use to move, I would like to use one antenna but in case I also have another one. (I am using python) 
help guys, I hope it does not disgust as an idea it is a project for university

**Comment by krichardsson** [comment]
You have to [change the address of one of your Crazyflies](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#firmware-configuration) (use the same channel on both Crazyflies).

If you changed the address to `E7E7E7E701` for instance, you can connect to both Crazyflies from the same script by using the two URIs, something like

`radio://0/80/2M/E7E7E7E7E7` and `radio://0/80/2M/E7E7E7E701`
